app/views/RoomsListView/index.js

```diff
@@ -328,6 +332,17 @@ export default class RoomsListView extends LoggedView {
 				this.removeListener(this.livechat);
 			}
 
+			// set Unread Count
+			this.data.addListener(debounce(() => {
```

Comment:
You should search for unread count in RoomView's componentDidMount.

Reply:
You want only unread count on componentDidMount?

Reply:
The headerBackTitle has to be defined in the origin screen, not in the destination screen. If I search for unread count in RoomView I don't can set that.

Reply:
I moved the listener to ComponentDidMount, is that?

Reply:
If you declare `renderLeft` on RoomViewHeader, you can pass the button title you want.
Querying on RoomsListView won't work on deep link/notifications tap and won't update automatically.

Reply:
Hello @diegolmello , in the version we are using of react-navigation the title prop of the HeaderBackButton isn't works, I have opened an issue in the react-navigation repository and I'm waiting for replies. Regards, Djorkaeff.